inst/tests/tests.Rraw

```diff
@@ -3106,6 +3106,15 @@ test(1017.4, fread(f, integer64="character"), fread(f, colClasses=c("integer", "
 
 unlink(f)
 
+f = tempfile()
+writeLines('a', f)
+writeLines(rep('0x1.ffffp0', 10000), f)
+writeLines('0x1.ff\x9fp0', f)
+writeLines(rep('0x1.ffffp0', 20000), f)
+fread(f)
```

Comment:
thanks!

please add a comment here describing the purpose of the test and cite the PR number.

also this should invoke test() with an associated number (which must be in increasing order).  maybe easiest to dump these lines at the bottom of the file instead.